extension/snippets/xcomponent.json

```diff
@@ -19,7 +19,7 @@
 		"prefix": "xctransition",
 		"description": "Create a XComponent transition",
 		"body": [
-			"<TransitionData Id=\"${1:Index}\" Name=\"${2:Transition name}\" FromKey=\"${3:State + Id}\" ToKey=\"${4:State + Id}\" Type=\"${5|X,TimeOut,Standard|}\" ExecutionDelay=\"${6:0}\" SetCustomTimeout=\"${7|false,true|}\" TriggeringEvent=\"${8:Document Triggering event}\" UserSpecificRule=\"${9|false,true|}\">",
+			"<TransitionData Id=\"${1}\" Name=\"${2:Transition name}\" FromKey=\"${3}\" ToKey=\"${4}\" Type=\"${5|X,TimeOut,Standard|}\" ExecutionDelay=\"${6:0}\" SetCustomTimeout=\"${7|false,true|}\" TriggeringEvent=\"${8:Document Triggering event}\" UserSpecificRule=\"${9|false,true|}\">",
```

Comment:
I wouldn't put non mandadory parameters like ExecutionDelay, SetCustomTimeout, UserSpecificRule inthe snippet.
Why not creating a snippet for each kind of transition?

Reply:
Good idea !
In this PR I was focused on intellisense (the snippet was created in a preview PR)
But it makes sense. I will update my PR with snippets for each kind of transition

Reply:
And fork transition.

Reply:
Yes of course

Reply:
done